CMakeLists.txt

```diff
@@ -841,15 +841,15 @@ if(CONFIG_LEGACY_GENERATED_INCLUDE_PATH)
     ${CMAKE_CURRENT_BINARY_DIR}/include/generated/syscall_list.h)
 endif()
 
-add_custom_command(OUTPUT include/generated/zephyr/syscall_dispatch.c ${syscall_list_h}
+add_custom_command(OUTPUT include/generated/zephyr/syscall_dispatch.c syscall_export_llext.c ${syscall_list_h}
   # Also, some files are written to include/generated/zephyr/syscalls/
   COMMAND
   ${PYTHON_EXECUTABLE}
   ${ZEPHYR_BASE}/scripts/build/gen_syscalls.py
   --json-file        ${syscalls_json}                     # Read this file
   --base-output      include/generated/zephyr/syscalls    # Write to this dir
   --syscall-dispatch include/generated/zephyr/syscall_dispatch.c # Write this file
-  --syscall-export-llext  include/generated/zephyr/syscall_export_llext.c
+  --syscall-export-llext syscall_export_llext.c
```

Comment:
why is this file no longer placed in `include/generated/zephyr` like the other Zephyr generated file ?

Reply:
Not a requirement, however I wanted to clearly show that unlike all other files in `include/generated` (AFAICS), this is directly compiled standalone to its own library. Let me know if I should revert this.